tests/contrib/hooks/test_gcp_translate_hook.py

```diff
@@ -0,0 +1,80 @@
+# -*- coding: utf-8 -*-
+#
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+
+import unittest
+
+from airflow.contrib.hooks.gcp_translate_hook import CloudTranslateHook
+from tests.contrib.utils.base_gcp_mock import mock_base_gcp_hook_default_project_id
+
+try:
+    from unittest import mock
+except ImportError:
+    try:
+        import mock
+    except ImportError:
+        mock = None
+
+PROJECT_ID_TEST = 'project-id'
+
+
```

Comment:
In L31, mock could be `None`, should we use in L35?
```suggestion
@unittest.skipIf(mock is None, 'mock package not present')
```
So as `test_gcp_translate_operator.py`

Reply:
This pattern is sprinkled all-over the Airflow code and copy&pasted everywhere. It's rather unlikely for mock to be None - in different python versions you simply expect it in either unittests or directly imported. I think this problem will be gone fairly soon when Airflow 2.0 drops python 2.7 compatibility, and until then I'd rather leave it as is.

Reply:
got